stereo_image_proc/launch/stereo_image_proc.launch.py

```diff
@@ -94,6 +92,10 @@ def generate_launch_description():
                     'left/image_rect_color',
                     [LaunchConfiguration('left_namespace'), '/image_rect_color']
                 ),
+                (
+                    'right/image_rect_color',
+                    [LaunchConfiguration('right_namespace'), '/image_rect_color']
+                ),
```

Comment:
this was a relevant patch from #720 (which was an alternative approach, before the overriding was really available)